.github/workflows/ci.yml

```diff
@@ -23,8 +23,6 @@ jobs:
       - uses: actions/checkout@v3
 
       - uses: guardian/setup-scala@v1
-        with:
-          major-java-version: 11
```

Comment:
Better to let `guardian/setup-scala` just to default to reading from the `.tool-versions` file so that Java version is defined in one less place!